pkg/kubectl/cmd/auth/cani.go

```diff
@@ -201,6 +205,9 @@ func (o *CanIOptions) Validate() error {
 		if o.Resource != (schema.GroupVersionResource{}) || o.ResourceName != "" {
 			return fmt.Errorf("NonResourceURL and ResourceName can not specified together")
 		}
+		if !isKnownNonResourceVerb(o.Verb) {
+			fmt.Fprintf(o.ErrOut, "Warning: verb %s is not a known verb\n", o.Verb)
```

Comment:
to make any whitespace issues clearer:

```suggestion
			fmt.Fprintf(o.ErrOut, "Warning: verb '%s' is not a known verb\n", o.Verb)
```